src/content/docs/en/guides/deploy/sst.mdx

```diff
@@ -6,50 +6,46 @@ i18nReady: true
 ---
 import { Steps } from '@astrojs/starlight/components';
 
-You can deploy an Astro site using [SST](https://sst.dev), an open-source framework for deploying fully serverless applications to AWS with SSG and SSR support.
+You can deploy an Astro site to AWS using [SST](https://sst.dev), an open-source framework for deploying modern full-stack applications with SSG and SSR support.
 
-You can also use any additional SST constructs like Cron Jobs, Buckets, Queues, etc while maintaining type-safety.
+You can also use any additional SST components like Cron Jobs, Buckets, Queues, etc while maintaining type-safety.
 
 ## Quickstart
 
 <Steps>
 1. Create an astro project.
 
-2. Run `npx create-sst`.
+2. Run `npx sst@latest init`.
 
 3. It should detect that you are using Astro and ask you to confirm.
 
-4. Once you're ready for deployment you can run `npx sst deploy --stage=production`.
+4. Once you're ready for deployment you can run `npx sst deploy --stage production`.
 </Steps>
 
-You can also watch [a video walkthrough of this process](https://www.youtube.com/watch?v=AFP3ZHxO7Gg) that will guide you through the steps.
+You can also read [the full tutorial](https://sst.dev/docs/start/aws/astro) that will guide you through the steps.
```

Comment:
```suggestion
You can also read [the full Astro on AWS with SST tutorial](https://sst.dev/docs/start/aws/astro) that will guide you through the steps.
```
It's important that link text be very clear about what it is and where it's taking you, especially when it's linking to an external site!